examples/multimodel/hand_face_person_detection_parallel_video_stream.ipynb

```diff
@@ -0,0 +1,147 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "65d4cd90",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# hw_location: where you want to run inference\n",
+    "#     @cloud to use DeGirum cloud\n",
+    "#     @local to run on local machine\n",
+    "#     IP address for AI server inference  \n",
+    "# model_zoo_url: url/path for model zoo\n",
+    "#     cloud_zoo_url: valid for @cloud, @local, and ai server inference options\n",
+    "#     '': ai server serving models from local folder\n",
+    "#     path to json file: single model zoo in case of @local inference\n",
+    "# hand_det_model_name: name of the model for hand detection\n",
+    "# face_det_model_name: name of the model for face detection\n",
+    "# person_det_model_name: name of the model for person detection\n",
+    "# video_source: video source for inference\n",
+    "#     camera index for local web camera\n",
+    "#     URL of RTSP stream\n",
+    "#     URL of YouTube Video\n",
+    "#     path to video file (mp4 etc) \n",
+    "hw_location='@cloud'\n",
+    "model_zoo_url = 'https://cs.degirum.com/degirum/public'\n",
+    "hand_det_model_name= 'yolo_v5s_hand_det--512x512_quant_n2x_orca1_1'\n",
+    "face_det_model_name= 'yolo_v5s_face_det--512x512_quant_n2x_orca1_1'\n",
+    "person_det_model_name= 'yolo_v5s_person_det--512x512_quant_n2x_orca1_1'\n",
+    "video_source='../../images/person_face_hand.mp4'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "290deaa8",
+   "metadata": {},
+   "source": [
+    "#### The rest of the cells below should run without any modifications"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "33012fe3",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Successfully opened video stream '../../images/person_face_hand.mp4'\n"
+     ]
+    }
+   ],
+   "source": [
+    "import degirum as dg, degirum_tools\n",
+    "# configure for Google Colab\n",
+    "degirum_tools.configure_colab() \n",
+    "# connect to AI inference engine getting token from env.ini file\n",
+    "zoo = dg.connect(hw_location, model_zoo_url, degirum_tools.get_token())\n",
+    "# load models for hand, face, and person detection\n",
+    "hand_det_model = zoo.load_model(hand_det_model_name)\n",
+    "face_det_model = zoo.load_model(face_det_model_name)\n",
+    "person_det_model = zoo.load_model(person_det_model_name, overlay_line_width = 1)\n",
+    "# AI prediction loop, press 'x' or 'q' to stop video\n",
+    "with degirum_tools.Display(\"Hands, Faces and Persons\") as display:\n",
+    "    for inference_result in degirum_tools.predict_stream(degirum_tools.CombiningCompoundModel(degirum_tools.CombiningCompoundModel(hand_det_model,\n",
```

Comment:
It would be nice to reformat all notebooks with Black formatter as well.
For example, this particular place has very misaligned formatting.

Reply:
Done.